app.js

```diff
@@ -943,44 +943,41 @@ define(function(require) {
 		prepareReviewData: function(data) {
 			var self = this,
 				expected = _.get(data, 'columns.expected', []),
+				modifierPerProp = {
+					brand: _.toLower,
+					family: _.toLower,
+					model: _.toLower
+				},
 				formattedData = {
 					data: {
 						fileName: data.fileName,
 						totalRecords: data.records.length,
 						columns: {
 							actual: data.columns.actual,
-							expected: expected
+							expected: expected,
+							others: []
 						},
-						recordsToReview: data.records.slice(0, 5),
+						recordsToReview: data.isDevices ? _.map(data.records, function(record) {
+							return _.chain(record)
+								.merge(record, _.reduce(record, function(row, prop, index) {
```

Comment:
Isn't `record` an object?
```suggestion
								.merge(record, _.reduce(record, function(acc, value, prop) {
```

Reply:
yeah, it is